src/interface/src/app/map/map.component.ts

```diff
@@ -547,18 +547,18 @@ export class MapComponent implements AfterViewInit, OnDestroy, OnInit, DoCheck {
     });
   }
 
-  /** Gets the selected region geojson and renders it on the map.
-   * Currently unused.
-   * */
-  private displayRegionBoundary(map: Map, selectedRegion: Region | null) {
-    if (!selectedRegion) return;
-    if (!map.instance) return;
-    this.mapService
-      .getRegionBoundary(selectedRegion)
-      .subscribe((boundary: GeoJSON.GeoJSON) => {
-        this.mapManager.maskOutsideRegion(map, boundary);
-      });
-  }
+  // /** Gets the selected region geojson and renders it on the map.
```

Comment:
So you may see this in some of the mocks, but originally when you would select a region an additional layer would get added to the map that would darken the "background", aka everything on the map outside of that region. That functionality currently works (possibly with a bug or two) because we have the assets for those layers locally loaded in, but there was some discussion on if it was better to try and have the geoserver load in those assets, which led to further discussions on whether or not the region boundaries themselves may change (I believe the local region layers highlight the rffc region boundaries, while the data we use is for the rrk region boundaries). Eventually we came to the decision to just remove region highlighting for now by disabling the function but leaving it in case y'all wanted to add it back in the future. You would be able to look at commit history on git if you ever wanted to reference this again, so you could remove it if you want. It may be worth discussing with the team if this is a feature y'all may add back in any time soon